waku/incentivization/reputation_manager.nim

```diff
@@ -0,0 +1,49 @@
+import tables
+import waku/waku_lightpush/rpc
+
+type
+  PeerId = string
+
+  ReputationIndicator* = enum
+    BadRep
+    GoodRep
+    UnknownRep
```

Comment:
Perhaps an `Option[bool]` would be more flexible than an enum, especially if we plan to eventually replace this with an analog reputation or integrate it as a weight in a peer score elsewhere? `Option[bool]` can be "none" (unknown), true (good?) or false (bad?).

Reply:
Fair enough. My reasoning for using a custom enum was mostly readability, but considering that it should be easier to integrate elsewhere with a simple type, I changed it to `Option[bool]` ([ecf94f6](https://github.com/waku-org/nwaku/pull/3293/commits/ecf94f629d5aff42cffb1f37a6249b685373085a)).